src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -12,7 +12,7 @@
     <Compile Include="CSharpArgumentInfoTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\src\Microsoft.CSharp.csproj">
+    <ProjectReference Include="..\pkg\Microsoft.CSharp.pkgproj">
```

Comment:
looks the VS experience is broken even without Karthik changes. I believe can load the solutions but I am seeing compiling the ref projects produce some errors and F5 experience to run the tests are broken too. 

I think we need to track this issue to look at as looks there is different problems to have the E2E scenarios work seamlessly.